utils/sparse.py

```diff
@@ -11,25 +11,36 @@
 
     wandb_available = True
 
+_LOGGER = logging.getLogger(__file__)
 
 def is_parallel(model):
     return type(model) in (
         nn.parallel.DataParallel, nn.parallel.DistributedDataParallel)
 
 
 class SparseMLWrapper(object):
-    def __init__(self, model, recipe):
+    def __init__(self, model, recipe, checkpoint_recipe=None, checkpoint_epoch=float('inf')):
         self.enabled = bool(recipe)
```

Comment:
i'd remove this auxiliary variable. non-empty string evaluates to True, so ` self.manager = ScheduledModifierManager.from_yaml(
            recipe
        ) if recipe else None` is fine imo.

Reply:
I feel this is a vanity change that maybe does not add anything extra, I will address this in a follow up, leaving it as is for now